src/components/DevModePanel/hooks.ts

```diff
@@ -0,0 +1,20 @@
+import { createContext, useContext } from "react";
+
+export const DevModeContext = createContext({
+  variables: "",
+  setVariables: (_value: string) => undefined,
+  isDevModeVisible: false,
+  setDevModeVisibility: (_value: boolean) => undefined,
+  devModeContent: "",
+  setDevModeContent: (_value: string) => undefined,
+});
```

Comment:
suggestion: create an interface describing context values
```suggestion
export const DevModeContext = createContext<null | YourInterfcae>(null)
```